cypress/integration/regressionTs/actionConditions.spec.ts

```diff
@@ -650,29 +679,105 @@ describe('Action Conditions', () => {
             })
         })
 
-        describe(`EnumConditions`, () => {
-            it(`when selecting different enum activities dependent on original enum should be marked with errors`, () => {
-                cy.get(s.trainDialogs.descriptions)
-                    .contains(testData.inputs.setEnumValue)
+        describe(`Condition Editing`, () => {
```

Comment:
Rest of changes are mostly whitespace, new test was for Condition Editing which this PR introduced